source/integrations/jira.rst

```diff
@@ -1,10 +1,17 @@
 .. _jira:
 
-JIRA Webhook Plugin 
+JIRA Plugin 
 ================================
 
 Set up a JIRA integration for your Mattermost instance within minutes using a pre-packaged JIRA webhook plugin.
 
+**Usage**
+
+Once enabled, JIRA posts notifications to a Mattermost channel when an issue is created, updated or deleted.
+
+.. image:: ../../source/images/jira_sample.png
+  :width: 300 px
```

Comment:
Propose increasing the width or removing it and letting it follow default size, it looks a little small at 300px
<img width="750" alt="screenshot 2018-10-05 17 32 14" src="https://user-images.githubusercontent.com/9354841/46561277-f6872f80-c8c4-11e8-82be-4fbd21d63e1a.png">